json/testnet-bitcoin-config.json

```diff
@@ -1,13 +1,13 @@
 {
   "bitcoinFeeSpendingCutoffPeriodInBlocks": 1,
   "bitcoinFeeSpendingCutoff": 0.001,
-  "bitcoinPeerUri": "http://localhost:18332",
+  "bitcoinPeerUri": "http://bitcoin-node:18332",
   "bitcoinRpcUsername": "",
   "bitcoinRpcPassword": "",
-  "bitcoinWalletOrImportString": "[FILL THIS IN!]",
+  "bitcoinWalletOrImportString": "cUuEua7N4cfCfjceHreEJ5k6TNtV9tko4q7hDwJ1t4qfvoAdjVQY",
```

Comment:
Should leave as it was?

Reply:
If this isn't set then the config that is baked into it for running the cluster of services in testnet wont work, we either have to decide between

1. No configs in the image at all and hence force the call to supplier their own when initializing the services 
2. Have a default config for testnet, like this approach is showing with a wallet that is auto refuelled by a faucet.

Reply:
I think option 2 does reduce friction for people experimenting,.